src/Command/Ingester/UserIngesterCommand.php

```diff
@@ -180,7 +180,7 @@ private function createUserDto(array $rawUser): UserDto
         return new UserDto(
             new UuidV6(),
             $rawUser['username'],
-            $this->passwordEncoder->encodePassword(new User(), $rawUser['password']),
+            $this->passwordEncoder->encodePassword(new User(new UuidV6(), 'notUsed', 'notUsed'), $rawUser['password']),
```

Comment:
Is this correct?
```
new User(new UuidV6(), 'notUsed', 'notUsed')
```
should be:
```
new User(new UuidV6(), $rawUser['username'], $rawUser['password'])
```
no?

Reply:
I just wanted to indicate that it's not used (it looks ugly I agree), fixed based on your recommendation.